esmvalcore/cmor/_fixes/cmip6/norcpm1.py

```diff
@@ -0,0 +1,21 @@
+"""Fixes for NorCPM1."""
+from ..fix import Fix
+
+
+class Nbp(Fix):
+    """Fixes for nbp."""
+
+    def fix_data(self, cube):
+        """
+        Fix data.
+        Parameters
+        ----------
+        cube: iris.cube.Cube
+        Returns
+        -------
+        iris.cube.Cube
+        """
+        metadata = cube.metadata
+        cube *= 29./44.
```

Comment:
Please add a comment explaining the fraction.

Reply:
@bettina-gier Could you please add this comment? I think you know why to multiply this fraction...